lib/sprig/reap/value.rb

```diff
@@ -0,0 +1,70 @@
+module Sprig::Reap
+  class Value
```

Comment:
Didn't write a unit test for this guy 'cause it's only got one public method that does all the work, and that's all covered by the unit test for `Sprig::Reap::Record.to_hash`.  Figured I'd end up re-writing basically the same tests here -- anyone have feels about this?